airflow/contrib/example_dags/example_gcp_automl_operators.py

```diff
@@ -0,0 +1,306 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that uses Google AutoML services.
+"""
+from typing import List, Dict
+import os
+from copy import deepcopy
+
+import airflow
+from airflow import models
+from airflow.contrib.hooks.gcp_automl_hook import get_object_id
+from airflow.contrib.operators.gcp_automl_operator import (
+    AutoMLTrainModelOperator,
+    AutoMLTablesListColumnSpecsOperator,
+    AutoMLPredictOperator,
+    AutoMLBatchPredictOperator,
+    AutoMLCreateDatasetOperator,
+    AutoMLTablesUpdateDatasetOperator,
+    AutoMLTablesListTableSpecsOperator,
+    AutoMLImportDataOperator,
+    AutoMLDeleteDatasetOperator,
+    AutoMLDatasetListOperator,
+    AutoMLDeployModelOperator,
+    AutoMLGetModelOperator,
+    AutoMLDeleteModelOperator,
+)
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "your-project-id")
+# For now only this location is supported
+GCP_AUTOML_LOCATION = os.environ.get("GCP_AUTOML_LOCATION", "us-central1")
+GCP_AUTOML_DATASET_BUCKET = os.environ.get("GCP_AUTOML_DATASET_BUCKET", "gs_source")
+TARGET = os.environ.get("GCP_AUTOML_TARGET", "Class")
+
+# Example values
+MODEL_ID = "TBL123456"
+DATASET_ID = "TBL123456"
+
+# Example model
+MODEL = {
+    "display_name": "auto_model_1",
+    "dataset_id": DATASET_ID,
+    "tables_model_metadata": {"train_budget_milli_node_hours": 1000},
+}
+
+# Example dataset
+DATASET = {
+    "display_name": "test_set",
+    "tables_dataset_metadata": {"target_column_spec_id": ""},
+}
+
+IMPORT_INPUT_CONFIG = {"gcs_source": {"input_uris": [GCP_AUTOML_DATASET_BUCKET]}}
+
+default_args = {"start_date": airflow.utils.dates.days_ago(1)}
+
+
+def get_target_column_spec(columns_specs: List[Dict], column_name: str) -> str:
+    """
+    Using column name returns spec of the column.
+    """
+    for column in columns_specs:
+        if column["displayName"] == column_name:
+            return get_object_id(column)
```

Comment:
```suggestion
            return extract_object_id(column)
```
It's better describe purpose of this method.

Reply:
What do you think about adding it to gcp_base_hook? It could be useful.

Reply:
Good idea, but we should introduce it in seperate PR.